### PR TITLE
[Bazel/MSVC] Enable png SIMD for MSVC

### PR DESCRIPTION
**Pull request by rongjiecomputer**

### CODE REVIEW

File: third_party/png.BUILD [hunk 1]
```diff
@@ -52,3 +63,8 @@ genrule(
     outs = ["pnglibconf.h"],
     cmd = "sed -e 's/PNG_ZLIB_VERNUM 0/PNG_ZLIB_VERNUM 0x12b0/' $< >$@",
 )
+
+config_setting(
+    name = "windows",
+    values = {"cpu": "x64_windows"},
+)
```
Comment (meteorcloudy) [inline third_party/png.BUILD]: Nit: new line please

**Review by meteorcloudy — DISMISSED** [review]

File: third_party/png.BUILD (hunk 1 above)
Comment (rongjiecomputer) [reply third_party/png.BUILD]: Done

**Comment by rongjiecomputer** [comment]
Windows Bazel builds all failed at link step: `LINK : fatal error LNK1000: unknown error at 000000013FE44241; consult documentation for technical support options`. Probably just flake.

MacOS Contrib also failed with mysterious "Internal CI infrastructure error".

Rebased against master, please trigger CI test again.

**Review by meteorcloudy — APPROVED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee @martinwicke: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Pull request merged**